python/graphscope/client/session.py

```diff
@@ -735,6 +735,8 @@ def close(self):
                     instance.close()
             except InteractiveEngineInternalError:
                 pass
+            except Exception:
+                pass
```

Comment:
Just catch `Exception` should be enough, as it is a baseclass of `InteractiveEngineInternalError `.